apps/web/src/server/applications/create/applications-create.service.js

```diff
@@ -69,12 +69,14 @@ export const createApplicationDraft = async (payload, session) => {
 
 /**
  * Get draft application by id
+ * optional query parameters can filter the returned data
  *
+ * @param {string | null} query
  * @param {number} id
  * @returns {Promise<Application>}
  */
-export const getApplicationDraft = async (id) => {
-	return get(`applications/${id}`);
+export const getApplicationDraft = async (id, query = null) => {
+	return get(`applications/${id}${query ? `?query=${query}` : ''}`);
```

Comment:
Do we need to default `query` to null? I would have thought that `query` defaults to `undefined` automatically?

Reply:
@DavidJWhiteDev - why does this equal null?

Reply:
there are existing calls to this that only have 1 argument.  Causes a syntax error.  Rather than changing all existing calls to add "..., null)" it is better to keep the default argument setting in the fn I think